README.md

```diff
@@ -37,9 +37,13 @@ Images are pushed to:
 - .NET 6.0
 - Go 1.20
 - JDK 11
-- Node.js 16
+- Node.js 18
 - Python 3.9
 
+### NodeJS LTS
+
+Our Docker images aim to use NodeJS LTS version (v18 as of writing). You can pin the image tag to a particular version in order to avoid unintended upgrades.
+
```

Comment:
Let's document the overall policy for all language runtimes, not just Node. I'll add to the commit.